### DISCUSSION TITLE
将 lint 结果输出到 html 感觉并不好用

### DISCUSSION THREAD

**Discussion by timongh** [the1812/Bilibili-Evolved#3201] [功能建议 / Ideas]
当前 vscode task 里的 lint 命令会将错误报告输出到一个 html 文件里面。但有两个问题：

1. 会失去 IDE 的跳转功能
2. 正确的结果非常多，错误的少。要在一堆正确结果中滚动到错误结果是一件非常麻烦的事情。

<img width="796" alt="Snipaste_2022-03-31_01-12-10" src="https://user-images.githubusercontent.com/46739861/160892844-2b7d687a-b1c5-43fe-b0c1-bdb3941902ae.png">

**Comment by the1812** [comment]
有道理, 我提供个非 html 的输出 task 吧
